routing/pathfind_test.go

```diff
@@ -1290,6 +1293,54 @@ func runPathFindingWithAdditionalEdges(t *testing.T, useCache bool) {
 	assertExpectedPath(t, graph.aliasMap, path, "songoku", "doge")
 }
 
+// runPathFindingWithRedundantAdditionalEdges asserts that we are able to
+// find paths to nodes ignoring additional edges that are already known by
+// self node.
+func runPathFindingWithRedundantAdditionalEdges(t *testing.T, useCache bool) {
+	graph, err := parseTestGraph(t, useCache, basicGraphFilePath)
```

Comment:
This test graph is mainly used in the older tests. At some point it was considered impractical because it ties all tests together.

Newer tests are based on `newPathFindingTestContext` that takes a set of channels directly. You could consider using that.

Reply:
This one was a breeze to add and looks way better. [a2079ec](https://github.com/lightningnetwork/lnd/pull/6766/commits/a2079ec9649152b86a637fb95f353634801382da)